### ISSUE TITLE
Loader._classloader_regressions_452707_b452707_b452707_ test failed in CI

### ISSUE THREAD

**Issue by stephentoub** [dotnet/coreclr#8894]
https://ci.dot.net/job/dotnet_coreclr/job/master/job/debug_windows_nt_prtest/1681/
```
D:\j\workspace\debug_windows---17180f1b\bin\tests\Windows_NT.x64.Debug\TestWrappers\Loader\Loader.XUnitWrapper.cs(21840): error : Loader._classloader_regressions_452707_b452707_b452707_._classloader_regressions_452707_b452707_b452707_cmd [FAIL] [D:\j\workspace\debug_windows---17180f1b\tests\runtest.proj]
        
  
  Return code:      1
  Raw output file:      D:\j\workspace\debug_windows---17180f1b\bin\tests\Windows_NT.x64.Debug\Reports\Loader\classloader\regressions\452707\b452707\b452707.output.txt
  Raw output:
  BEGIN EXECUTION
         "D:\j\workspace\debug_windows---17180f1b\tests\..\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root\corerun.exe" b452707.exe 
        Expected: 100
        Actual: -1
        END EXECUTION - FAILED
        FAILED
        Test Harness Exitcode is : 1
        
  To run the test:
  > set CORE_ROOT=D:\j\workspace\debug_windows---17180f1b\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root
  > D:\j\workspace\debug_windows---17180f1b\bin\tests\Windows_NT.x64.Debug\Loader\classloader\regressions\452707\b452707\b452707.cmd
  
        Expected: True
        Actual:   False
          D:\j\workspace\debug_windows---17180f1b\bin\tests\Windows_NT.x64.Debug\TestWrappers\Loader\Loader.XUnitWrapper.cs(21840,0): at Loader._classloader_regressions_452707_b452707_b452707_._classloader_regressions_452707_b452707_b452707_cmd()
    Finished:    Loader.XUnitWrapper
    Finished:    GC.XUnitWrapper
    Finished:    JIT.XUnitWrapper
  === TEST EXECUTION SUMMARY ===
     baseservices.XUnitWrapper  Total:    8, Errors: 0, Failed: 0, Skipped: 0, Time:   13.022s
     CoreMangLib.XUnitWrapper   Total:    1, Errors: 0, Failed: 0, Skipped: 0, Time:   10.774s
     Exceptions.XUnitWrapper    Total:    2, Errors: 0, Failed: 0, Skipped: 0, Time:   10.769s
     GC.XUnitWrapper            Total:  608, Errors: 0, Failed: 0, Skipped: 0, Time:  665.818s
     Interop.XUnitWrapper       Total:   43, Errors: 0, Failed: 0, Skipped: 0, Time:   78.851s
     JIT.XUnitWrapper           Total: 5757, Errors: 0, Failed: 0, Skipped: 0, Time: 2423.584s
     Loader.XUnitWrapper        Total:  471, Errors: 0, Failed: 1, Skipped: 0, Time:  558.796s
     managed.XUnitWrapper       Total:    1, Errors: 0, Failed: 0, Skipped: 0, Time:  105.424s
     readytorun.XUnitWrapper    Total:    5, Errors: 0, Failed: 0, Skipped: 0, Time:   27.615s
     Regressions.XUnitWrapper   Total:   18, Errors: 0, Failed: 0, Skipped: 0, Time:   44.699s
                                       ----          -          -           -        ---------
                          GRAND TOTAL: 6914          0          1           0        3939.351s (2453.975s)
```

**Comment by gkhanna79** [comment]
@stephentoub Looking at recent jobs for this at https://ci.dot.net/job/dotnet_coreclr/job/master/job/debug_windows_nt_prtest/, I dont see this failure anymore. 

If you concur, should we close this?

**Comment by stephentoub** [comment]
It was a random failure hit by an unrelated PR, suggesting some kind of race condition somewhere (though I've not looked at the test). I don't believe lack of it happening since means the issue went away.

**Comment by gkhanna79** [comment]
Was there a specific reason to make you suspect a race here (as opposed to a regression that got fixed in a PR update)?

**Comment by stephentoub** [comment]
Because I found no other issues related to it, this test hasn't changed since it was checked in almost a year ago, I saw no other failures of the test before or after, and in my experience all of that suggests unreliability due to either employed randomness or a race condition somewhere (src, test, infrastructure, wherever) that caused a spurious failure.

**Comment by jeffschwMSFT** [comment]
@stephentoub this is a pretty old issue.  I took a look today at the test and it is dreadfully simple (calls a valuetype that overrides ToString).  The comment in the test indicates that this behavior had regressed in the past due to a precode issue.  I ran this test in a loop (10k+) on a ret and dbg version of coreclr and it passed.  Was this test disabled?  If so I think it would be worth re-enabling it.
